packages/maker/deb/test/MakerDeb_spec.ts

```diff
@@ -27,7 +27,9 @@ describe('MakerDeb', () => {
 
   beforeEach(() => {
     ensureFileStub = stub().returns(Promise.resolve());
-    eidStub = stub().resolves();
+    const eidPromise = Promise.resolve();
+    (eidPromise as any).transformVersion = require('electron-installer-debian').transformVersion;
+    eidStub = stub().returns(eidPromise);
```

Comment:
I'm having a difficult time trying to get it so that the `require('electron-installer-debian')` stub has the real `transformVersion` attached to it.

Reply:
That doesn't seem to work, I get `TypeError: installer.transformVersion is not a function`. I think there's something weird with proxying going on.

Reply:
Very strange, it appears to work in CI but not locally.